cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -489,6 +489,14 @@ func doInstall(mst *initramfsMountsState, model *asserts.Model, sysSnaps map[sna
 		return err
 	}
 
+	// Now we can write the snapd mount unit (needed as this is the first boot)
+	isRunMode := false
```

Comment:
I suppose it does not matter because install does not happen for classic. But technically we are is some run mode if we do install in initrd.

Reply:
Right, it is sort of debatable. I've added a comment.